lib/rspecq/worker.rb

```diff
@@ -119,9 +128,19 @@ def try_publish_queue!(queue)
       RSpec.configuration.files_or_directories_to_run = files_or_dirs_to_run
       files_to_run = RSpec.configuration.files_to_run.map { |j| relative_path(j) }
 
+      config = {
+        'fail_fast': @fail_fast
```

Comment:
With the new logic, we can just do in `#initialize`:

```ruby
# ...
@queue = Queue.new(.....)
@queue.fail_fast = fail_fast
```

Then we wouldn't need to pass anything to `#publish` or construct a config options hash.

Reply:
fail_fast is a method in Queue